### DISCUSSION TITLE
Add multiple images and text to a PDF

### DISCUSSION THREAD

**Discussion by spectre3brad** [Stirling-Tools/Stirling-PDF#2384] [Ideas]
Hello everyone,

I would like to know if it was possible to include the possibility of opening a PDF and inserting several images and text.

For example: I have a PDF document to which I would like to add a certification stamp, a handwritten signature and the name of the signatory.

Would it be possible to include it later in this solution?

THANKS.

**Comment by Frooodle** [comment]
does the Sign PDF option not offer this functionality?

**Comment by spectre3brad** [comment]
Perfect, I wasn't using the software well and hadn't seen certain options. It works, thank you.
